app/lib/actions/candlepin/candlepin_listening_service.rb

```diff
@@ -59,7 +59,7 @@ def start(suspended_action)
         else
           suspended_action.notify_not_connected("Not Connected")
         end
-      rescue TransportFailure => e
+      rescue => e
```

Comment:
@ehelms oh there it is, thanks. We are seeing alot of issues with uninitialized constants in rails 4, is there a way to reference TransportFailure from a top level?